src/app/shared/form-elements/language-select/language-select.component.ts

```diff
@@ -9,7 +9,7 @@ import {LanguageSelectItemService} from './language-select-item.service';
 })
 export class LanguageSelectComponent {
   @Input()
-  label: string = 'Language';
+  label: string | null = null;
```

Comment:
I think we're a little inconsistent here. I saw another instance where the string has been initialized as an empty string instead of null.

We should probably decide on one